### ISSUE TITLE
Text in Unicode bold/italics/etc is not recognized by the search

### ISSUE THREAD

**Issue by mgttlinger** [rogershen/chrome-regex-search#30]
If you try to search for `test` here then only the regular variant will be found. The normal search of Chrome handles this case as one would expect.

regular: test
bold serif: 𝐭𝐞𝐬𝐭
bold sans: 𝘁𝗲𝘀𝘁
italics serif: 𝑡𝑒𝑠𝑡
italics sans: 𝘵𝘦𝘴𝘵
bold italic serif: 𝒕𝒆𝒔𝒕
bold italic sans: 𝙩𝙚𝙨𝙩

**Comment by limeb** [comment]
please fix this sub and sup don't work either

**Comment by mtoconno** [comment]
Unicode works for me in Chrome

**Comment by skeets23** [comment]
@mtoconno - you're saying when you use this extension on this page to search "test", it gets all 8 instances in the top post here?

**Comment by skeets23** [comment]
note: It looks like if I copy Japanese text and paste it directly into the search, it works.

However, if I type the Japanese directly into the regex search box, it comes up with zero results.

**Comment by rogershen** [comment]
Yes, I can confirm this is a problem. Unicode definitely complicates things and I admittedly don't really know how to solve it. I found this article on the topic: https://mathiasbynens.be/notes/javascript-unicode. If you guys can find a good solution, I would be happy to take a pull request.
